cmd/options/options.go

```diff
@@ -118,6 +128,15 @@ func (npdo *NodeProblemDetectorOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&npdo.PrometheusServerAddress, "prometheus-address",
 		"127.0.0.1", "The address to bind the Prometheus scrape endpoint.")
 
+	for _, exporterName := range exporters.GetExporterNames() {
+		fs.StringSliceVar(
+			npdo.ExporterConfigPaths[exporterName],
+			"exporter."+string(exporterName),
+			[]string{},
+			fmt.Sprintf("Comma separated configurations for %v exporter. %v",
```

Comment:
For exporters, we just need one config file per exporter, right? Or should we enforce that? Do you expect we would have multiple stackdriver exporters?

Reply:
Good point. Changed the code to only access one config file per exporter.